script.js

```diff
@@ -0,0 +1,59 @@
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+function displayBooks() {
+  const booksDiv = document.getElementById('books');
+  booksDiv.innerHTML = '';
+
+  if (books.length === 0) {
+    booksDiv.textContent = 'No books in the collection.';
+    return;
+  }
+
+  const removeBook = (index) => {
+    books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+    displayBooks();
+  };
+
+  books.forEach((book, index) => {
+    booksDiv.innerHTML += `
+    <div class="book">
+        <span class="book-title">${book.title}</span>
+        <span class="book-author">by ${book.author}</span>
+        <button class="remove-button" type="button" data-index="${index}">Remove</button>
+      </div>`;
+  });
+
+  const removeButtons = document.getElementsByClassName('remove-button');
+  Array.from(removeButtons).forEach((button, index) => {
+    button.addEventListener('click', () => {
+      removeBook(index);
+    });
+  });
+}
+
+function addBook(title, author) {
+  const book = {
+    title,
+    author,
+  };
```

Comment:
- [OPTIONAL] I highly recommend adding unique identifiers (e.g., `id` field) to new books for easy modification or removal. Improves functionality and maintainability.